examples/rpc_send_example.c

```diff
@@ -145,6 +144,10 @@ main(int argc, char **argv)
         goto cleanup;
     }
 
+    /* RPC output */
+    sr_val_t *output = NULL;
+    size_t output_count = 0;
+
```

Comment:
Seems there is a problem with you moving this definition because these variables are used in `cleanup`, which can be jumped to even before they are defined. Please keep the definition where it was.

Reply:
Done. I never use goto, thus my confusion

Reply:
One should not overuse it, but it is often handy especially for error handling.